providers/standard/src/airflow/providers/standard/sensors/bash.py

```diff
@@ -24,7 +24,12 @@
 from typing import TYPE_CHECKING
 
 from airflow.exceptions import AirflowFailException
-from airflow.sensors.base import BaseSensorOperator
+from airflow.providers.standard.version_compat import AIRFLOW_V_3_0_PLUS
```

Comment:
I think it also means that we need to add `>=NEXT_VERSION` for the providers in the main `pyproject.toml` for the extras for Airflow 3.1

The providers that are already released will start failing on Airflow 3.1.

Reply:
(ah no... Wrong... we have deprecation)...